configuration/multimedia.md

```diff
@@ -386,3 +384,109 @@ listenAndAnswer(null, null)
+
+```text
+increase the <item name>
+decrease the <item name>
+set the color of the <item name> to red
+put the <item name> to next
+put the <item name> to previous
+play the <item name>
+pause the <item name>
+rewind the <item name>
+fast forward the <item name>
+start the <item name>
+stop the <item name>
+refresh the <item name>
+```
+
+For exact built-in grammar you can refer to the [source code](https://github.com/openhab/openhab-core/blob/main/bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/internal/text/StandardInterpreter.java#L42).
+
+
+#### Target item
+
+The interpreter resolves the item name based on the its label/synonyms and it's parent label/synonyms.
+
+An example of the possible situations will be:
+
+If you have `Group` item labeled as `TV` with a `Dimmer` child `Brightness`.
+The interpreter understands these phrases as the same `turn off tv`, `turn off brightness`, `turn off tv brightness` and `turn off brightness tv`.
+
+If you add a Switch child labeled as `Power` to the group.
+The interpreter understands now also the phrases `turn off power`, `turn off tv power` and `turn off power tv`.
+But the `turn off tv` phase now detect a collision because two matching items accept the `OFF` command.
+
+#### Name prevalence
+
+One way you can solve this is by using the name prevalence, items with start with other items names take prevalence over them.
+
+If the Switch item has the name `tv` and the and Dimmer item the name `tv_brightness` there will be no collisions between them and the `OFF` command will target the Switch item.
+
+#### Exact match label/synonym prevalence
+
+Other way you can solve this is by using the exact match prevalence, items with have an exact match on their label/synonyms take prevalence.
+
+If the Switch item has the synonym `TV` there will be no collisions between them and the `OFF` command will target the Switch item.
+
+#### Location prevalence
+
+The dialog processor forward its locationItem to the standard interpreter to be used for reducing collisions on the target resolution.
+
+If you have two items labeled as `Light` but one is a child of the location item you have configured for the dialog execution, the item takes prevalence.
+
+The location takes prevalence over an exact match.
+
+#### Item description rules
+
+The interpreter also creates rules for your item descriptions for English, German, French, Italian and Spanish.
+
+If you have a Dimmer item called `Light` with command description `100=high` the interpreter will also understand the phrase `Set light to high`.
+
+### Item custom rules.
+
+You can register custom rules into the interpreter using items and the metadata namespace `voiceSystem`.
+
+The examples of valid rules:
+
+```
+"start? watch|watching $*$ on $name$":
+Matches "start watching some show on tv" sends command "some show".
```

Comment:
Maybe ->
Matches "start watching some show on tv" and sends command "some show".